src/Raven.Server/Documents/Handlers/Debugging/DocumentDebugHandler.cs

```diff
@@ -54,5 +58,65 @@ public async Task HugeDocuments()
                 writer.WriteEndObject();
             }
         }
+        
+        [RavenAction("/databases/*/debug/documents/scan-corrupted-ids", "GET", AuthorizationStatus.ValidUser, EndpointType.Read, IsDebugInformationEndpoint = true)]
```

Comment:
`IsDebugInformationEndpoint` does it mean that during debug package creation we will hit this EP?